doc/classes/XRServer.xml

```diff
@@ -50,6 +58,19 @@
 				Finds an interface by its [param name]. For example, if your project uses capabilities of an AR/VR platform, you can find the interface for that platform by name and initialize it.
 			</description>
 		</method>
+		<method name="get_face_tracker" qualifiers="const">
+			<return type="XRFaceTracker" />
+			<param index="0" name="tracker_name" type="StringName" />
+			<description>
+				Returns the [XRFaceTracker] with the given tracker name.
+			</description>
+		</method>
+		<method name="get_face_trackers" qualifiers="const">
+			<return type="Dictionary" />
+			<description>
+				Returns a dictionary of the registered [XRFaceTracker] names to instances.
```

Comment:
May be worth being more specific. E.g. "Each element of the dictionary is a _ corresponding to a _" or "Each key is a _ corresponding to _", etc..

Reply:
Done.